zomes/hc_zome_trust_atom/tests/trust_atom_tests.rs

```diff
@@ -413,7 +413,8 @@ pub async fn test_get_extra() {
     .await;
```

Comment:
let's have a test that pushes in 1000 bytes of content, and: 
- ensure that we get an extra hash in the link tag
- retrieve the content from the extra entry and validate that it is correct